src/templates/blog-post.js

```diff
@@ -34,16 +34,34 @@ const PostHeader = ({ author, post }) => {
 }
 
 const PostFeaturedImage = ({ post }) => {
-  const featuredImgFluid = post.frontmatter?.featuredImage?.childImageSharp?.fluid
-
-  if (!featuredImgFluid) return (null);
+  const featuredImage = post.frontmatter?.featuredImage?.childImageSharp?.fluid
```

Comment:
I've never actually seen/noticed the "?" in JS expressions before - handy!
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
Yeah those were something new to me too; don't think I'll ever not use them now :D